src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/OffenderDomainEventListener.kt

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.digital.hmpps.personrecord.message.listeners
```

Comment:
Very minor but I note that the above class is called CourtCaseEvent**s**Listener  and this one is OffenderDomainEventListener - I feel we should go for either "Event" or "Events" and stick with it

Reply:
'Events' is the correct one here as well. Will update it in the next PR.Thanks.